201-ospatching-extension-on-ubuntu/prereqs/prereq.azuredeploy.json

```diff
@@ -37,7 +37,7 @@
     },
     "storageAccountKey": {
       "type": "string",
-      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2016-01-01').keys[0].value]"
+      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value]"

```

Comment:
I'm going to merge this but wanted to see if you want to grab it... we shouldn't be outputting the keys, but the resourceId, and then in the template where the key is needed, perform listkeys... so the secret isn't visible in the deployment.